aiohttp/client_reqrep.py

```diff
@@ -82,6 +88,10 @@ def __init__(self, method, url, *,
         self.url = url.with_fragment(None)
         self.original_url = url
         self.method = method.upper()
+
+        if self.method in self.GET_METHODS and any((data, compress, chunked)):
+            raise ValueError('Method %s should not contain request-data.' % self.method)
```

Comment:
OK, I didn't know that ElasticSearch predates RFC, where it's said that HTTP-server SHOULD ignore payload of GET-requests. I'm very upset about everything that.